yarn.lock

```diff
@@ -1529,6 +1529,30 @@
     npmlog "^4.1.2"
     write-file-atomic "^3.0.3"
 
+"@mattrglobal/bbs-signatures@0.4.0", "@mattrglobal/bbs-signatures@^0.4.0":
+  version "0.4.0"
+  resolved "https://registry.yarnpkg.com/@mattrglobal/bbs-signatures/-/bbs-signatures-0.4.0.tgz#de106c4a5946e590dc1ecaf672e90c284a5fba85"
+  integrity sha512-r6ed3N9xO+7AlO6FF2rVksH+HjfLwxZ9eKWHg3saTi5hzhJyehE3Qahj+goddgYD6emzwzrVbawNZDTttMQUKg==
+  optionalDependencies:
+    "@mattrglobal/node-bbs-signatures" "0.10.0"
+
+"@mattrglobal/bls12381-key-pair@^0.4.1-unstable.256739c":
+  version "0.4.1-unstable.f581696"
+  resolved "https://registry.yarnpkg.com/@mattrglobal/bls12381-key-pair/-/bls12381-key-pair-0.4.1-unstable.f581696.tgz#989c9b918baa1b26a2db6ac3a90115eb9d7f8c9f"
+  integrity sha512-EtOK60//gsmZjtzumdN08pfa/41tSpY1XNrwKhxq3fv0z3a7Tna2+HflnohhUQOGH7xpvpd+aUkzDm/OFpMsaA==
+  dependencies:
+    "@mattrglobal/bbs-signatures" "0.4.0"
```

Comment:
Not completely sure why a signature package depends on a Key package. We should probably report that to Mattr (should be the other way round)